janusgraph-dist/src/assembly/static/conf/janusgraph.env

```diff
@@ -0,0 +1,4 @@
+JANUSGRAPH_YAML=/opt/janusgraph/conf/gremlin-server/gremlin-server.yaml
```

Comment:
For the rpm and deb implementations I was planning to use `/etc/opt/janusgraph` to match linux standards and the docker image. The problem is that the server yamls have relative paths so changing the directories for these files will make things a little harder on users if they choose to manually setup.

In `janusgraph-docker` there is just one properties file named `janusgraph.properties`. I think it would be cleaner to do the same and just document that users should copy over whichever template they want to use. It might work best to make copies of the properties and yaml files with updated paths that just get used by rpm and deb packages. It's really import to get this detail right and I'll give this some more thought. 

I'm thinking of a structure like this right now

```
/etc/opt/janusgraph/janusgraph.properties
/etc/opt/janusgraph/janusgraph-server.yaml
/etc/opt/janusgraph-script.groovy
/etc/opt/janusgraph/conf/*.yaml
/etc/opt/janusgraph/conf/*.properties
```
From with the `/etc/opt/janusgraph` directory users could change templates like `cp conf/janusgraph-cassandra-es.properties janusgraph.properties`

Reply:
A point I failed to address was using `JANUSGRAPH_CFG` instead would be much better for this. I still need to test out how to fix the issues of the paths within the gremlin-server.yaml file.

Reply:
Why `/etc/opt/janusgraph` instead of `/etc/janusgraph`?

Normalizing and simplifying the file names in the config directory would be good.

* graph.properties
* server.yaml
* server.groovy

Of course, this is assuming `JanusGraphFactory` as opposed to `ConfiguredGraphFactory`, in which case the script file might not be as necessary with the dynamic graph bindings.

Reply:
I asked the same thing in the initial janusgraph-docker pr and got schooled in this [comment](https://github.com/JanusGraph/janusgraph-docker/pull/1#pullrequestreview-151053377)

Here's the link to the [spec from the linuxfoundation](http://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s07.html#etcoptConfigurationFilesForOpt)
My main concern is to be consistent across the board.

Reply:
@pluradj I just pushed an additional commit with `/etc/opt/janusgraph` as the `JANUSGRAPH_CFG` default. Since `gremlin-server.yaml` doesn't care about `JANUSGRAPH_CFG` I had to hardcode the new paths for the properties and script files in a new template file, `daemon-gremlin-server.yaml`